org.testeditor.web.backend.persistence/src/main/java/org/testeditor/web/backend/persistence/workspace/WorkspaceResource.xtend

```diff
@@ -26,6 +26,8 @@ class WorkspaceResource {
 
 	private def WorkspaceElement createWorkspaceElements() {
 		val git = gitProvider.git
+		git.pull.call
```

Comment:
I think the pull should be made within `listFiles()` and not here. We could pass in the `Git` instance as a parameter in order to avoid getting it twice.